presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -355,3 +360,75 @@ Lambda Aggregate Functions
         -- (2, 42)
 
     The state type must be a boolean, integer, floating-point, or date/time/interval.
+
+
+Filtering During Aggregation
```

Comment:
Move this to the top, since it is describing generic syntax and shouldn't be mixed in between the descriptions of specific functions. Also, let's add a related section heading
```
Ordering During Aggregation
---------------------------